### PR TITLE
Feature/consent request form UI

### PR DESCRIPTION
**Pull request by norbert989**
# Consent request form UI

Added styling and markup for the consent request form.

- To test this log in to PB
- Navigate to the Privacy Page
- Click the bottom left mock link

TODO: Add missing icons to prism and update as part of the logic implementation ticket
# Checklist

- [x] All acceptance criteria are met.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
Might be worth populating the request page with mock data based on the API doc instead of hard coding strings (might need updating later but we can have a rough idea of what the consent request would look like).

Also adding some tests for the interactions even if for now is just checking that a handler is called.

**Comment by VirginiaBalseiro** [comment]
Also, why did you choose [rid] instead of just [id] for the consent id in the Consent Show page?

**Review by VirginiaBalseiro — COMMENTED** [review]
Left a couple comments

File: components/pages/privacy/consent/requests/styles.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      minHeight: 50,
+    },
+
+    "request-container__section": {
+      padding: 0,
+      border: 0,
+      width: "100%",
+      borderRadius: 4,
+      display: "flex",
+      flexGrow: "1",
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/requests/styles.js:50]: This doesn't need to be a string.

File: components/pages/privacy/consent/requests/styles.js [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      padding: theme.spacing(0.6),
+    },
+
+    "request-container__header-text": {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-between",
+      alignItems: "baseline",
+      fontSize: "1.4rem",
+      margin: "0",
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/requests/styles.js:88]: Same here, this should be a number.

File: components/pages/privacy/consent/requests/styles.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+      fontFamily: theme.typography.body2.fontFamily,
+    },
+
+    "request-container__header-text--center": {
+      textAlign: "center",
+    },
+
+    "icon-small": {
+      ...theme.icons.iconColor(theme.palette.info.main),
+      fontSize: 16,
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/requests/styles.js:103]: 16 is the default, so I think this is not needed. In any case 1rem should result in 16.

File: components/pages/privacy/consent/styles.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+      minHeight: 50,
+    },
+
+    "request-container__section": {
+      padding: 0,
+      border: 0,
+      width: "100%",
+      borderRadius: 4,
+      display: "flex",
+      flexGrow: "1",
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/styles.js:51]: Same here, this should be a number.

File: components/pages/privacy/consent/styles.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+      padding: theme.spacing(0.6),
+    },
+
+    "request-container__header-text": {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-between",
+      alignItems: "baseline",
+      fontSize: "1.4rem",
+      margin: "0",
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/styles.js:89]: Same here, this should be a number.

File: components/pages/privacy/consent/styles.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      fontFamily: theme.typography.body2.fontFamily,
+    },
+
+    "request-container__header-text--center": {
+      textAlign: "center",
+    },
+
+    "icon-small": {
+      ...theme.icons.iconColor(theme.palette.info.main),
+      fontSize: 16,
```
Comment (VirginiaBalseiro) [inline components/pages/privacy/consent/styles.js:104]: Same as above.

File: components/consentRequestForm/styles.js [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+      minHeight: 50,
+    },
+
+    "request-container__section": {
+      padding: 0,
+      border: 0,
+      width: "100%",
+      borderRadius: 4,
+      display: "flex",
+      flexGrow: "1",
```
Comment (VirginiaBalseiro) [inline components/consentRequestForm/styles.js:50]: This doesn't need to be a string, should be a number.

File: components/consentRequestForm/styles.js [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+      padding: theme.spacing(0.6),
+      whiteSpace: "nowrap",
+    },
+
+    "request-container__header-text": {
+      display: "flex",
+      flexDirection: "row",
+      alignItems: "baseline",
+      fontSize: "1.4rem",
+      margin: "0",
```
Comment (VirginiaBalseiro) [inline components/consentRequestForm/styles.js:88]: Same here, this should be a number.

**Review by VirginiaBalseiro — CHANGES_REQUESTED** [review]
Nice work! Just left a few comments.

File: components/consentRequestForm/styles.js (hunk 7 above)
Comment (VirginiaBalseiro) [reply components/consentRequestForm/styles.js:50]: This one seems to be unresolved

**Review by VirginiaBalseiro — APPROVED** [review]
One tiny thing was left from previous review

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by kevin-inrupt — APPROVED** [review]

**Pull request merged**